build/networking/README_DOCKER.md

```diff
@@ -77,226 +75,122 @@ container up and running, and should see the container prompt like below:
 
 `c5efb1a949ad` is your container id as displayed in command `docker ps -a`
 
-As your P4-OvS should be up and running, you could see the ovsdb-server
-and ovs-vswitchd process with the `ps -ef | grep ovs` command.
-
-## Section 2: Running example setup ( VM1 <-> IPDK Container <-> VM2 )
-Below commands will help you setup traffic between 2 VMs on your host with the
-IPDK container as a P4 program enabled vswitch switching traffic between them.
-
-Pre-requisite:
-- Container in Section 1 should be up and with OvS running
-- If you are running in a VM then make sure you have nested virtualization
-  enabled on your guest VM
-- QEMU and KVM should be installed and running
+## Run Networking Recipe
+Below commands will help you setup the environment for Networking recipe and
```

Comment:
Using `Networking recipe` here while above uses `Networking Recipe`. We should keep it consistent. Are we going with uppercase `R`?